components/lvgl/CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
 file(GLOB_RECURSE SOURCES lvgl/src/*.c)
 idf_component_register(SRCS ${SOURCES}
-                       INCLUDE_DIRS . lvgl)
+                       INCLUDE_DIRS . lvgl lvgl/src/lv_themes)
```

Comment:
Why `lv_themes` is special?

Reply:
When testing with monochrome displays the monochrome theme wasn't found, I think I can solve it bu using `#include "lvgl/src/lv_themes"` in the main file?

Reply:
Fixed with b88fcbd